metricbeat/helper/prometheus/prometheus.go

```diff
@@ -66,6 +70,16 @@ func (p *prometheus) GetFamilies() ([]*dto.MetricFamily, error) {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode > 399 {
+		if p.logger.IsDebug() {
+			bodyBytes, err := ioutil.ReadAll(resp.Body)
+			if err == nil {
+				p.logger.Debug("error received from prometheus endpoint: ", string(bodyBytes))
```

Comment:
do you expect this body to be large? as this error is fairly important we could consider showing it by default. With some trimming if needed

Reply:
don't expect large errors payloads there.
will change it to show description without debug enabled